Common/GraphicsContext.h

```diff
@@ -30,7 +30,10 @@ class GraphicsContext {
 
 	// Called from the render thread from threaded backends.
 	virtual void ThreadStart() {}
-	virtual bool ThreadFrame() { return true; }
+	virtual bool ThreadFrame() {
+        // Needs to return false if no queue draining is required.
```

Comment:
nit: Sorry, there's some indentation mismatches.  I recommend installing an editorconfig plugin in your editor.

https://editorconfig.org/

-[Unknown]